### PR TITLE
Recommend byteSize typed as a non-negative integer (not decimal)

### LINKED ISSUE #125: dcat:byteSize - check constraints
In DCAT v1 the property `dcat:byteSize` is axiomatized 
```
dcat:byteSize
  rdf:type owl:DatatypeProperty ;
  rdfs:domain dcat:Distribution ;
  rdfs:range rdfs:Literal ;
.
```

1. Verify that the range is appropriate and necessary
2. Verify that the domain is appropriate and necessary (see #110)
3. Consider whether any guarded constraints (using owl:Restriction) should be introduced (see #105)

### PR DESCRIPTION
**Pull request by dr-shorthair**
Addressing #125

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -2703,8 +2703,9 @@ <h4>Property: byte size</h4>
                 <tbody>
                 <tr><td class="prop">Definition:</td><td>The size of a distribution in bytes.</td></tr>
                 <tr><td class="prop">Domain:</td><td><a href="#Class:Distribution"><code>dcat:Distribution</code></a></td></tr>
-                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a> typed as <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a>.</td></tr>
-                <tr><td class="prop">Usage note:</td><td>The size in bytes can be approximated (as a decimal) when the precise size is not known.</td></tr>
+                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a> typically typed as <a data-cite="XMLSCHEMA11-2#integer"><code>xsd:integer</code></a>.</td></tr>
```
Comment (andrea-perego) [inline dcat/index.html]: If we are going to change it, then probably we should go for `xsd:positiveInteger` or `xsd:nonNegativeInteger`.

**Comment by riccardoAlbertoni** [comment]
I have  expressed a concern related to back-compatibility,  see https://github.com/w3c/dxwg/issues/125#issuecomment-811857000

Am I over-worrying?

**Review by riccardoAlbertoni — APPROVED** [review]

**Comment by agbeltran** [comment]
AFAICT the changed requested by @andrea-perego, and I agree with, has not been implemented - happy to take care of it

**Comment by riccardoAlbertoni** [comment]
> AFAICT the changed requested by @andrea-perego, and I agree with, has not been implemented - happy to take care of it

Fine to me, Thanks.
 I would use `xsd:positiveInteger' .  See https://github.com/w3c/dxwg/issues/125#issuecomment-818660079

**Comment by riccardoAlbertoni** [comment]
Just a reminder, we might need to update the examples in the specification that use `xsd:decimals`  for `dcat:byteSize` and the notes/definition in the RDF files for DCAT.

**Comment by andrea-perego** [comment]
> Just a reminder, we might need to update the examples in the specification that use `xsd:decimals` for `dcat:byteSize` and the notes/definition in the RDF files for DCAT.

AFAICS, there are 2 examples to be updated:
- https://w3c.github.io/dxwg/dcat/#ex-distribution
- https://w3c.github.io/dxwg/dcat/#ex-when-using-distribution

**Comment by agbeltran** [comment]
@dr-shorthair @andrea-perego @riccardoAlbertoni as discussed, I've changed the range to ```xsd:nonNegativeInteger``` and udpated the examples accordingly

**Comment by agbeltran** [comment]
Change history also updated

**Review by riccardoAlbertoni — APPROVED** [review]

File: dcat/index.html [hunk 2]
```diff
@@ -6449,6 +6450,9 @@ <h2>Changes since the first public working draft of 17 December 2020</h2>
 
 <p>The document has undergone the following changes since the DCAT 3 first public working draft of 17 December 2020 [[?VOCAB-DCAT-3-20201217]]:</p>
 
+<ul>
+  <li><p>The range of property <a href="#Property:distribution_size"></a> has been changed from <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> to <a data-cite="XMLSCHEMA11-2#nonNegativeInteger"><code>xsd:nonNegativeInteger</code></a> </p></li>
```
Comment (andrea-perego) [inline dcat/index.html]:
```suggestion
  <li><p>The recommended range of property <a href="#Property:distribution_size"></a> has been changed from <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a> to <a data-cite="XMLSCHEMA11-2#nonNegativeInteger"><code>xsd:nonNegativeInteger</code></a> </p></li>
```

**Review by andrea-perego — APPROVED** [review]

**Comment by agbeltran** [comment]
Merging now - issue remains open in case there are more comments

**Pull request merged**